packages/gatsby-plugin-graphql-codegen/webpack.config.js

```diff
@@ -0,0 +1,40 @@
+// const nodeExternals = require('webpack-node-externals')
+
+module.exports = {
+  entry: './src/gatsby-node.ts',
+  target: 'node',
+  output: {
+    filename: 'gatsby-node.js',
+    path: __dirname,
+    libraryTarget: 'commonjs'
+  },
+  resolve: {
+    extensions: [".ts", ".tsx", ".js"]
+  },
+  externals: [
+    {
+      'fs-extra': 'commonjs2 fs-extra',
```

Comment:
What does putting `commonjs` before the package here do? I've never seen this before.

Reply:
It's additional context so webpack understand that the module (`fs-extra`) is a commonjs module, [docs here](https://webpack.js.org/configuration/externals/#externals). It's only useful for bundling node stuff. Without `commonjs2`, webpack would look for `fs-extra` in global space